pkg/restclient/cluster.go

```diff
@@ -0,0 +1,22 @@
+package restclient
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gluster/glusterd2/pkg/api"
+)
+
+// GetClusterOption gets cluster level options
+func (c *Client) GetClusterOption() ([]api.ClusterOptionsResp, error) {
```

Comment:
add e2e test case of this.

Reply:
Not a part of this PR, this PR only addresses glustercli commands, I will add e2e test cases for cluster option in separate PR, please raise an issue for it will fix that issue.

Reply:
why you would like to add Client library in one PR and add e2e in other PR. what the concern here?
even if you are adding a functionality for CLI, you are adding a client library.